src/odemis/driver/test/actuator_test.py

```diff
@@ -1042,7 +1042,12 @@ def setUp(self):
                                           refproc="Standard",
                                           )
 
-        self.dev_cycle = RotationActuator("stage", "stage", {self.axis_name: self.dependency1}, self.axis, ref_start=1)
+        self.dev_cycle = RotationActuator("stage", "stage", 
+                                          dependencies={self.axis_name: self.dependency1},
+                                          axis_name=self.axis,
+                                          ref_start=1,
+                                          ref_on_move_count=7
+                                         )
```

Comment:
Did you check the testcases where the referencing is checked? There were certain things checked if they are true after 5 moves, as we expected the referencing there. Now that it is not hardcoded anymore, we might want to update the tests as well. See `test_offset_moveAbs`.

Reply:
I hadn't noticed the test cases. Thanks for noticing it. I've now updated them.